### DISCUSSION TITLE
Change handler signature from (res, req) to ({ res, req })

### DISCUSSION THREAD

**Discussion by rtritto** [uNetworking/uWebSockets.js#1110] [Ideas]
Improve DX of `handler` changing function signature:
```diff
- app.get('/', (req, res) => ...)
+ app.get('/', ({ req, res }) => ...)
```

Object parameter of `handler` becomes the _context_.

Some advantages:
- migration from `express` (and similar or modern libraries) are easier (order isn't relevant)
- you can omit _req_ or _res_ if not used in the _handler_
- in the future, other properties can be added to _context object_ as _features_ and not as _breaking changes_

**Comment by uNetworkingAB** [comment]
You're focusing on irrelevant differences. There are way more fundamental differences that could be fixed. Like #1036
